statscache_plugins/releng/__init__.py

```diff
@@ -164,12 +166,13 @@ def initialize(self, session):
             except Exception as e:
                 log.exception(
                     "Error during initializing releng plugin '{}': {}".format(
-                        plugin.idx, e), exc_info=True)
+                        plugin.ident, e), exc_info=True)
 
     def cleanup(self):
         pass
 
     def load_plugins(self, config, model):
+        # TODO: plugins for a plugin? This shoudln't be necessary
```

Comment:
Ha!  Yeah.  They had some common features to share, but the "plugins on plugins" model may not have been the right fit.  ;)